CHANGES.md

```diff
@@ -1,5 +1,54 @@
 # Changes
 
+## stable-2.8.0
+
+This release introduces new a multi-cluster extension to Linkerd, allowing it
+to establish connections across Kubernetes clusters that are secure,
+transparent to the application, and work with any network topology.
+
+* The CLI has a new set of `linkerd multicluster` subcommands that provide
+  tooling to create the resources needed to discover services across
+  Kubernetes clusters.
+* The `linkerd multicluster gateways` command exposes gateway-specific
+  telemetry,  addition to the existing `stat` and `tap` commands.
+* The Linkerd-provided Grafana instance remains enabled by default, but it can
+  now be disabled; and the Linkerd dashboard can be configured to link to an
```

Comment:
Don't think `and` needs to follow after a semicolon. A new sentence could also just work here.

```suggestion
  now be disabled; the Linkerd dashboard can be configured to link to an
```